nginx/default.toml

```diff
@@ -19,6 +19,12 @@ worker_processes = 4
 # worker_connections: Connections per Worker Process.  Default = 1024
 worker_connections = 1024
 
+[[include.additional_configuration_file]]
+config_file = "mime.types"
+
+[[include.additional_configuration_file]]
+config_file = "win-utf"
```

Comment:
I don't see why.  In my view, this package has way too many immutable defaults. Like this, `mime.types` and `win-utf` are now optional, right?